mne/io/persyst/tests/test_persyst.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+# Authors: Adam Li  <adam2392@gmail.com>
+#          simplified BSD-3 license
+
+import os.path as op
+import shutil
+
+import pytest
+from numpy.testing import assert_array_equal
+
+import mne
+from mne.datasets.testing import data_path, requires_testing_data
+from mne.io import read_raw_persyst
+from mne.utils import run_tests_if_main
+
+fname_lay = op.join(
+    data_path(download=False), 'Persyst',
+    'sub-pt1_ses-02_task-monitor_acq-ecog_run-01_clip2.lay')
+fname_dat = op.join(
+    data_path(download=False), 'Persyst',
+    'sub-pt1_ses-02_task-monitor_acq-ecog_run-01_clip2.dat')
```

Comment:
you should be calling the _test_raw_reader function on you reader to make sure it's ok.